### PR TITLE
Add David Dowell as codeowner

### PR DESCRIPTION
**Pull request by christinaholtNOAA**
## DESCRIPTION OF CHANGES: 
Add David Dowell to codeowners list

### CODE REVIEW

**Review by terraladwig — APPROVED** [review]

**Comment by christinaholtNOAA** [comment]
I will wait for David's go-ahead before I merge this one. I suggested he also be included here, but he didn't explicitly request it.

**Comment by daviddowellNOAA** [comment]
I'm OK with being added as a codeowner.

**Pull request merged**
